packages/side-nav/src/SideNav.tsx

```diff
@@ -35,11 +161,116 @@ interface SideNavProps {
  * @param props.className Class name that will be applied to the root-level element.
  * @param props.children Content that will be rendered inside the root-level element.
  */
-function SideNav({ className, children, ...rest }: SideNavProps) {
+function SideNav({ className, children, id: idProp, ...rest }: SideNavProps) {
+  const { Provider: ContextProvider } = SideNavContext;
+  const [collapsed, setCollapsed] = useState(false);
```

Comment:
Should we offer collapsed as a prop, such that we can have a side nav that is always open?

Reply:
Or `collapsible` rather

Reply:
No – we want behavior across apps to be universally consistent. That's one of the problems we currently have.

Reply:
That's fair, but maybe we can offer something like defaultOpen so that you can control if its collapsed or not on first render?

Reply:
I don't want that behavior to be inconsistent across implementations. We should be the ones controlling that, not the consumer.